lib/ca_testing/drivers/v4/options.rb

```diff
@@ -30,22 +23,13 @@ def initial_options(browser)
             case browser
             when :chrome;             then ::Selenium::WebDriver::Chrome::Options.new
             when :firefox;            then ::Selenium::WebDriver::Firefox::Options.new(log_level: "trace")
-            when :edge;               then edge_options
+            when :edge;               then ::Selenium::WebDriver::Edge::Options.new
             when :safari;             then ::Selenium::WebDriver::Safari::Options.new(automatic_inspection: true)
             when :internet_explorer;  then internet_explorer_options
             else {}
             end
           end
 
-          # This is only needed before version 4.0 is stable
```

Comment:
The EdgeHtml options (v3), is actually "more" stable than the EdgeChrome in v4 alpha.

EdgeChrome became the default in the rc1 release (But by then had some bug fixes), hence this plaster for all working perfectly.